### ISSUE TITLE
Failing tests on maven 3.8.5

### ISSUE THREAD

**Issue by laeubi** [eclipse/tycho#726]
The following integration tests fail on maven 3.8.5
```
[ERROR] Errors: 
[ERROR]   MultiplatformProductTest.exportProduct:27 » Verification Error in execution: [...
[ERROR]   PomDependencySystemScopedTest.testSystemScopedDependenciesIgnored:32 » Verification
[ERROR]   TestNGBundleTest.test:32 » Verification Error in execution: [ERROR] 'dependenc...
[ERROR]   Tycho154BundleJarTest.test:26 » Verification Error in execution: [ERROR] 'depe...

```

**Comment by laeubi** [comment]
The full error is for example:

Error in execution: [ERROR] 'dependencies.dependency.systemPath' for org.eclipse.tycho.tycho-its.tycho-154:bundle:eclipse-plugin:simple.jar is missing. @
	at org.eclipse.tycho.test.tycho154.Tycho154BundleJarTest.test(Tycho154BundleJarTest.java:26)

so probably this is a tycho issue.

**Comment by mickaelistria** [comment]
If this is not a Tycho issue, then is it a Maven issue?

**Comment by laeubi** [comment]
Haven investigated yet any further, we might can have a gh action for maven 3.8.5 as described on the malinglist by Mikael Barbero:

> curl -sSJOLf https://dist.apache.org/repos/dist/dev/maven/maven-3/3.8.5/binaries/apache-maven-3.8.5-bin.tar.gz
> $ tar zxf apache-maven-3.8.5-bin.tar.gz 
> $ export PATH="$(pwd)/apache-maven-3.8.5/bin:${PATH}"
> $ mvn -v 

https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners

but the eror message suggest that a systempath is not set, so maybe maven has ignored this before and now print it as an error ...

**Comment by mickaelistria** [comment]
Could those failures be related to this regression in Maven: https://issues.apache.org/jira/browse/MNG-7432 ?

**Comment by laeubi** [comment]
Maybe yes, maybe no :-)
I'm currently a bit short in time, maybe I can try that out next week, at laest there is a Quickfix so if you check that out and compile a local maven you could try if that makes a difference.

**Comment by mickaelistria** [comment]
This seems fixed thanks to your extra commit for #789
